.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "omnisharp.defaultLaunchSolution": "Npgsql.sln",
+    "dotnet-test-explorer.testProjectPath": "**/*.Tests.csproj"
+}
```

Comment:
Oh, you did it! Thank you!

One thing I would like to have is a styling setting in `.editorconfig` for JSON files, so all of them will use two spaces for indentation.

Reply:
Done.